src/components/homepage/homepagetemplate.js

```diff
@@ -24,21 +33,27 @@ const styles = theme => ({
     fontSize: '45px',
     lineHeight: '45px',
   },
+  banner: {
+    maxWidth: '100%',
+  },
 });
 
 class HomePage extends React.Component {
-  render() {
+  render () {
     const { classes } = this.props;
     return (
-      <div className={classes.container}>
-        <img className={classes.banner} src={banner}/>
-        <div className={classes.title}>
-          <div className={classes.mainTitle}>Move Fast.</div>
-          <div className={classes.mainTitle}>Build Things.</div>
-          <div className={classes.subTitle}>Start Hacking.</div>
+      <React.Fragment>
```

Comment:
Reminder: we could just return an array of elements.

Reply:
if we just return an array here, wouldn't that require us to put a `key` on the elements? does that make sense to put a `key` though?

Reply:
I don't think `key` is required but let's just stick with fragment then.